src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -332,13 +333,15 @@ class DiskImageEditor extends Component {
                 }
                 { !createMode && msg.diskEditorResizeLabel() }
               </LabelCol>
-              <Col sm={9}>
-                <FormControl
-                  type='number'
-                  value={this.state.values.size}
-                  onChange={this.changeSize}
-                />
-              </Col>
+              <OverlayTooltip id={`${idPrefix}-resize-tooltip`} tooltip={msg.diskEditorSizeTooltip()} placement='right'>
```

Comment:
Maybe better to wrap in `OverlayTooltip` only `FormControl`?

Reply:
why?

Reply:
> why?

Because the tooltip is for the FormControl itself and not the entire column.